docs/modules/ROOT/pages/api/erc721.adoc

```diff
@@ -252,7 +268,7 @@ NOTE: Implementing xref:api/introspection.adoc#SRC5Component[SRC5Component] is a
 * xref:#IERC721-Transfer[`++Transfer(from, to, token_id)++`]
 --
 
-==== Embeddable functions
+==== Embeddable Functions
```

Comment:
Why this change? I think it should be lowercase, but if we use uppercase, we should update the rest for consistency.

Reply:
All of the API pages except for ERC721 and ERC20 have `Embeddable Functions` in title case, so now this heading is at least in a consistent casing. See [Access](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/modules/ROOT/pages/api/access.adoc?plain=1#L86), [Account](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/modules/ROOT/pages/api/account.adoc?plain=1#L135), [Introspection](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/modules/ROOT/pages/api/introspection.adoc?plain=1#L73), and [Security](https://github.com/OpenZeppelin/cairo-contracts/blob/main/docs/modules/ROOT/pages/api/security.adoc?plain=1#L36). It's hard to confirm if there are other instances like this haha I can change the embeddable fns back if you prefer lower case

Reply:
We have "Internal functions" below (in all those cases), so the issue is that we are being inconsistent already. I think we should change "Embeddable Functions" to "Embeddable functions" for consistency with our title casing.